src/lang/es-ES.lang.json

```diff
@@ -151,7 +156,51 @@
         },
         "lock-timeline": "Fijar navegación a la línea temporal",
         "unlock-timeline": "Desbloquear navegación",
-        "translation-by": "Traducido por"
+        "translation-by": "Traducido por",
+        "image-credits": "by {credits}",
+        "share": "Share",
+        "measurement": {
+            "button": "Measure distances",
+            "title": "Measurement",
+            "explanation": "Click any two points to calculate their positions and the distance between them.\n\nNote that the Rosharan degree (°ᴿ) ranges from 0 to 200.",
+            "labels": {
+                "start": "Start",
+                "end": "End",
+                "distance": "Distance"
+            },
+            "coordinates": {
+                "full": "{latitude}, {longitude}",
+                "north": "{degrees}°ᴿ N",
+                "south": "{degrees}°ᴿ S",
+                "east": "{degrees}°ᴿ E",
+                "west": "{degrees}°ᴿ W"
+            },
+            "unit": "imperial",
```

Comment:
Hello @PaleoCrafter , I'm trying to translate this into Chinese. May I ask what's this `unit` used for? I searched in source code with keyword `measurement.unit`, but found nothing matched.

Reply:
Hi @lianzhao, good catch! That was left over from when I only had the measurements show one unit of distance (either miles or kilometers). But we're displaying both now so it's not used anymore. `measurement.distance` is the relevant thing now :)

Reply:
Got it. I'll remove this line from `zh.lang.json` as well. Thank you @PaleoCrafter .